packages/devtools/lib/src/timeline/flame_chart_canvas.dart

```diff
@@ -481,11 +484,14 @@ class FlameChartNode {
     canvas.fillText(
       displayText,
       rect.left + horizontalPadding,
-      rect.top + textOffsetY,
+      rect.top + _textOffsetY,
+      rect.width - horizontalPadding * 2, // maxWidth
```

Comment:
any time you write a comment like this it is a sign you should create a local variable instead. E.g.
final maxWidth = rect.width - horizontalPadding * 2;